install.sh

```diff
@@ -0,0 +1,48 @@
+# Constants
+VERSION="v0.4.2"
+SIMPLEX_CHAT="simplex-chat"
+
+######################
+PLATAFORM=""
+
+if [ "$(uname)" == "Darwin" ]; then
+	PLATAFORM="macos-x86-64"
+elif [ "$(uname)" == "Linux" ]; then
+	PLATAFORM="ubuntu-20_04-x86-64"
+else
+	echo "Your platform is not suported, try with macos/linux."
+	exit 1
+fi
+
+PLATAFORM_BIN="$SIMPLEX_CHAT-$PLATAFORM"
+######################
+
+######################
+# Detect the shell and the file where to export the path
+FILE_TO_EXPORT=""
+
+if [ -n "$($SHELL -c 'echo $ZSH_VERSION')" ]; then
+	FILE_TO_EXPORT=".zshrc"
+elif [ -n "$($SHELL -c 'echo $BASH_VERSION')" ]; then
+	FILE_TO_EXPORT=".bashrc"
```

Comment:
I think `~/.profile` would be a more appropriate file for expanding PATH, see an explanation here [https://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path/26059#26059](https://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path/26059#26059) "Where to put it". Probably this part doesn't even have to be shell-dependent, but better double check.

Reply:
Now I'm not exporting anything so I don't need to know the shell.

Reply:
@efim-poberezkin the previous version was better, as zsh (default on Mac) does not use .profile, it uses .zprofile instead, and most tools I know add lines to .zshrc and .bashrc (not sure why, but maybe they are reserving .profile/.zprofile to what is edited by a user manually...)

@UltiRequiem I will revert the commit that changed if it's ok?